cli/src/test/java/org/jboss/sbomer/cli/test/integ/feature/sbom/client/SBOMerClientTestIT.java

```diff
@@ -137,11 +163,35 @@ void testSearchSbomRequest() {
             try {
                 Page<SbomGenerationRequestRecord> sbomRequests = objectMapper.readValue(json, typeReference);
                 assertNotNull(sbomRequests);
-                assertEquals("AABBCC", sbomRequests.getContent().iterator().next().id());
+                assertEquals(id, sbomRequests.getContent().iterator().next().id());
                 assertEquals("QUARKUS", sbomRequests.getContent().iterator().next().identifier());
             } catch (JsonProcessingException e) {
                 fail(e.getMessage());
             }
         }
     }
+
+    @Test
+    void testNetworkConnectionTimeout() {
+
+        // Check for ultimate success first
+        testSearchSbomRequestParamerterised("AABBCCDD", "id==AABBCCDD");
+
+        // Check what actually happened after
+        // Print the requests for visual confirmation
+        List<ServeEvent> servs = wireMockServer.getAllServeEvents();
+        for (ServeEvent se : servs) {
+            log.info(
+                    "Request URL: " + se.getRequest().getUrl() + ", Method: " + se.getRequest().getMethod()
+                            + ", Status: " + se.getResponse().getStatus() + ", Matched Stub Id: "
+                            + se.getStubMapping().getId() + ", Matched Stub Name: " + se.getStubMapping().getName());
```

Comment:
This should use `{}` instead of `+`.

Reply:
Agreed, it wasn't showing useful information either other than the request url, so i've swapped a few things around to help in future if needed